lizard_connector/query.py

```diff
@@ -0,0 +1,158 @@
+import datetime
+import urllib.parse
+
+from lizard_connector import jsdatetime
+
+
+class LizardApiImproperQueryError(Exception):
+    pass
+
+
+class QueryDictionary(dict):
+
+    def update(self, E=None, *queries, **f):
```

Comment:
Dit is een kopie met kleine uitbreiding van de Python3 dict update functie. Daarom deze rare variabelen.